src/main/java/com/internship/frejaeidjmeterplugin/jmeter/visualizer/gui/ResultPanel.java

```diff
@@ -0,0 +1,117 @@
+
+package com.internship.frejaeidjmeterplugin.jmeter.visualizer.gui;
+
+public class ResultPanel extends javax.swing.JPanel {
+    private int delivered;
+    private int failed;
+    
+    public ResultPanel() {
+        initComponents();
+        delivered = 0;
+        failed = 0;
+    }
+    
+   public void setError (String error){
+       lblError.setText(error);
+   } 
+    
+   public void increaseDelivered (){
+        delivered+=1;
+        lblDelivered.setText(delivered+"");
+    }
+   
+    public void increasetFailed (){
+        failed+=1;
+        lblFailed.setText(failed+"");
+    }
+
+    public void clearGui (){
+        delivered = 0;
+        failed = 0;
+        lblDelivered.setText("");
+        lblFailed.setText("");
+    }
+    
+    @SuppressWarnings("unchecked")
+    // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
+    private void initComponents() {
+
+        jLabel1 = new javax.swing.JLabel();
+        lblDelivered = new javax.swing.JLabel();
+        jLabel3 = new javax.swing.JLabel();
+        lblFailed = new javax.swing.JLabel();
+        jButton1 = new javax.swing.JButton();
+        lblError = new javax.swing.JLabel();
+
+        jLabel1.setText("Delivered:");
+
+        lblDelivered.setText(" ");
+
+        jLabel3.setText("Failed:");
+
+        jButton1.setText("Restart values");
+        jButton1.addActionListener(new java.awt.event.ActionListener() {
+            public void actionPerformed(java.awt.event.ActionEvent evt) {
+                jButton1ActionPerformed(evt);
+            }
+        });
+
+        lblError.setFont(new java.awt.Font("Tahoma", 0, 12)); // NOI18N
+        lblError.setForeground(new java.awt.Color(255, 51, 51));
+
+        javax.swing.GroupLayout layout = new javax.swing.GroupLayout(this);
+        this.setLayout(layout);
+        layout.setHorizontalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
```

Comment:
Move this code to private method and describe what it is used for, same for vertical group